File: Cointln/cointl.pl
#!/usr/bin/perl
use strict;
use warnings;
use CGI;

my $q = CGI->new;
print $q->header(-type => 'text/plain', -charset => 'utf-8');

# Recibir el número desde la URL
my $numero = $q->param('n') // 0;

# Diccionario local para la conversión nativa (Rango básico de prueba)
my @unidades = ('cero', 'uno', 'dos', 'tres', 'cuatro', 'cinco', 'seis', 'siete', 'ocho', 'nueve', 'diez');

if ($numero >= 0 && $numero <= 10) {
    print $unidades[$numero];
} else {
    print "Número fuera de rango para la prueba local en Perl. Intenta del 0 al 10.";
}

File: Cointln/cointl.js
// conintl.js - Localización Nativa en Node.js
const express = require('express');
const app = express();

// Algoritmo local nativo para convertir números a palabras en español sin depender de internet
function numeroALetras(num) {
    const unidades = ['cero', 'uno', 'dos', 'tres', 'cuatro', 'cinco', 'seis', 'siete', 'ocho', 'nueve', 'diez'];
    return unidades[num] || "Número fuera de rango de prueba";
}

app.get('/', (req, res) => {
    const numero = parseInt(req.query.n || 0);
    res.send(numeroALetras(numero));
});

app.listen(8003, () => console.log("Servidor conintl (Node.js) activo en http://localhost:8003"));